dix/ut/client_ut.c

```diff
@@ -2531,15 +2531,16 @@ static void local_failures(void)
 	rc = dix_common_idx_op(&index, 1, REQ_CREATE);
 	M0_UT_ASSERT(rc == 0);
 	/*
-	 * Only two CAS requests can be sent successfully, but N + K = 3, so
-	 * no record will be successfully put to all component catalogues.
+ 	 * Consider DIX request to be successful if there is at least
+ 	 * one successful CAS request. Here two cas requests can be 
+ 	 * sent successfully. 
 	 */
 	m0_fi_enable_off_n_on_m("cas_req_replied_cb", "send-failure", 2, 3);
 	rc = dix_ut_put(&index, &keys, &vals, 0, &rep);
 	m0_fi_disable("cas_req_replied_cb", "send-failure");
 	M0_UT_ASSERT(rc == 0);
 	M0_UT_ASSERT(rep.dra_nr == COUNT);
-	M0_UT_ASSERT(m0_forall(i, COUNT, rep.dra_rep[i].dre_rc != 0));
+	M0_UT_ASSERT(m0_forall(i, COUNT, rep.dra_rep[i].dre_rc == 0));
```

Comment:
The comment above says two cas requests 'can' be successful. 
However, here the check validates that all COUNT replies are successful, is this correct?

Reply:
Each cas request have 10 KV pairs. Here it was validated 10 KV records. (COUNT means 10).